doc/source/examples/tools/data_input.ipynb

```diff
@@ -12,7 +12,9 @@
   },
```

Comment:
I don't like that the notebooks are now somewhere hidden in the docs source. They should still reside in some examples folder in the top dir, for users to quickly get started with some examples.

Btw you also have to rewrite the Readme and links in there

Reply:
Me neither, but this is actually a [big issue with `nbsphinx`](https://github.com/spatialaudio/nbsphinx/issues/170). I have tried several other solutions, but this is required especially for rendering the cell output.

Reply:
Puh. But they can be copied right? Its not pretty but I would prefer that as a workaround instead of moving them there

Reply:
Yes, that's the [alternative 'solution'](https://github.com/spatialaudio/nbsphinx/issues/170#issuecomment-373497587).

Reply:
So what are your arguments against this alternative solution? I can only think of some github links to the source file not matching at the end.

Reply:
I'm not against the alternative solution.

Reply:
https://github.com/ComPWA/pycompwa/pull/74/commits/f756a57b88adaf7a55910c779e74cb6f379bf179